src/database/queries.ts

```diff
@@ -82,6 +86,58 @@ export const getVoucher = async (uuid: uuid): Promise<Voucher> => {
   }
 };
 
+// TODO: Convert ENUM to actual type
+export const createVoucher = async (voucher: VoucherCreate): Promise<uuid> => {
+  try {
+    const docRef = await addDoc(voucherCollection, voucher);
+    console.log('Document ref id: ', docRef.id);
+    return docRef.id;
```

Comment:
We want to make sure the uuid is also a field in the firebase document.

`await updateDoc(docRef, { uuid: docRef.id });`

This line takes the generated `id` field and adds it as a field in the document.

<img width="438" alt="image" src="https://user-images.githubusercontent.com/21160510/197112432-0a94f44f-edd3-4a13-a91b-d2e969c09601.png">